app/konnect/gateway-manager/data-plane-nodes/transit-gateways.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to configure Transit Gateway
+---
+
+
+This guide will walk you through connecting your {{site.konnect_short_name}} managed Dedicated Cloud Gateways to AWS Transit Gateway, providing a secure and private channel for your API traffic.
+
+## Prerequsites 
+
+Before you start setting up the Transit Gateway with your {{site.konnect_short_name}} Dedicated Cloud Gateways, ensure that you have the following prerequisites:
```

Comment:
```suggestion
```
I think you could remove this preamble since it's implied that they have to have these before doing the steps.